docs/reference/config.rst

```diff
@@ -167,20 +167,38 @@ Defaults to ``yes``.
 list_format_item
 ~~~~~~~~~~~~~~~~
 
+Deprecated option, replaced by :ref:`format_item`.
+
+.. _list_format_album:
+
+list_format_album
+~~~~~~~~~~~~~~~~~
+
+Deprecated option, replaced by :ref:`format_album`.
```

Comment:
I tried leaving just the labels but `:ref:\`list_format_item\`` would turn into "format_album" (take the title below its definition) and that did not seem desirable. If you'd prefer it I'll remove those subsections.